android/app/build.gradle

```diff
@@ -27,23 +27,27 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
     namespace 'be.weforza.app'
-    compileSdkVersion 33
+    compileSdkVersion 34
```

Comment:
Since Flutter 3.13 supports Android 14, I've bumped our compilation target to support it as well.